vowpalwabbit/core/src/simple_label.cc

```diff
@@ -55,6 +55,22 @@ void VW::details::return_simple_example(VW::workspace& all, void*, VW::example&
   VW::finish_example(all, ec);
 }
 
+void VW::details::record_stats_simple_label(
```

Comment:
maybe `update_stats` instead of `record_stats`? it is in sync with `sd.update` and it is clear that it has side effects on a shared object since record could just mean record the stats somewhere

Reply:
`update_stats` is better I agree